Formula/libshout.rb

```diff
@@ -31,24 +33,8 @@ class Libshout < Formula
     depends_on "openssl@1.1"
   end
 
-  # libshout's libtool.m4 doesn't properly support macOS >= 11.x (see
-  # libtool.rb formula). This causes the library to be linked with a flat
-  # namespace which might cause issues when dynamically loading the library with
-  # dlopen under some modes, see:
-  #
-  #  https://lists.gnupg.org/pipermail/gcrypt-devel/2021-September/005176.html
-  #
-  # We patch `configure` directly so we don't need additional build dependencies
-  # (e.g. autoconf, automake, libtool)
-  #
-  # Patch has been submitted upstream:
-  # https://gitlab.xiph.org/xiph/icecast-libshout/-/issues/2332
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-pre-0.4.2.418-big_sur.diff"
```

Comment:
Have you tried updating patch to https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-big_sur.diff rather than using `autoreconf`?